client/src/components/Landing.js

```diff
@@ -2,14 +2,22 @@ import React from "react";
 import { Link } from "react-router-dom";
 
 const Landing = () => {
+  const buttonStyle = {
+    backgroundColor: "#6059F6", padding: "10px", border: "none", color: "white"
+   };
+
+   const jumbotronStyle = {
+     textAlign: "center"
+    };
+
   return (
-    <div className="jumbotron mt-5">
-      <h1>Welcome to Todo City</h1>
-      <p>Sign In and start building your todo list</p>
-      <Link to="/login" className="btn btn-primary">
+    <div className="jumbotron mt-5" style={jumbotronStyle}>
+      <h1>WELCOME TO TRACKR</h1>
+      <p>Join us in the first step to finding your dream job</p>
```

Comment:
"the first step" sounds weird.  Start your journey to find your dream job with us?